server/api/db/sqldb/db.py

```diff
@@ -899,15 +900,30 @@ def tag_artifacts(
         artifacts,
         project: str,
     ):
+        artifacts_keys = [artifact.key for artifact in artifacts]
+        logger.debug(
+            "Locking artifacts in db before tagging artifacts",
+            project=project,
+            tag=tag_name,
+            artifacts_keys=artifacts_keys,
+        )
```

Comment:
I know this is debug log, but it still is too verbose for production code...

Reply:
@theSaarco Since we are locking the DB I think it will be beneficial for us to leave these logs here.